tasks/prerequisite.yml

```diff
@@ -21,8 +21,13 @@
 - debug:
     msg: "{{ ss_utility }}"
 
-- name: 'Installing required packages like hp-scripting-tools and "{{ ss_utility }}"'
-  package: name={{ item }} state=latest
-  with_items:
-    - hp-scripting-tools
-    - "{{ ss_utility }}"
+- name: Install requirement packages
+  package:
+    name: "{{ item.name }}"
+    enablerepo: "{{ item.repo }}"
+    state: latest
+  with_items: 
+    - {name: "{{ ss_utility }}", repo: spp}
```

Comment:
Testing this on one of our HP compute nodes
```
failed: [fqdn.of.the.node]  (item={'name': 'hpssacli', 'repo': 'spp'}) => {"ansible_loop_var": "item", "changed": false, "item": {"name": "hpssacli", "repo": "spp"}, "msg": "Repository spp not found."}
failed: [fqdn.of.the.node] (item={'name': 'hp-scripting-tools', 'repo': 'stk'}) => {"ansible_loop_var": "item", "changed": false, "item": {"name": "hp-scripting-tools", "repo": "stk"}, "msg": "Repository stk not found."}
```

Reply:
We have kept repository configurations in separate role on purpose. In Lumi-O environment we have used these configuratons. https://github.com/CSCfi/lumi-o/blob/master/roles/common/tasks/hp_repo_setup.yml

Reply:
What do you wish to achieve by being able to run tasks without defining the repos? Also firmware to install are to be downloaded from HP repositories.

Reply:
Do you mean to run task as it was before my changes? If I remember correct, the original task failed because it did not have a repository for the packages to install. What do you wish to do with this role without repos?

Reply:
our hp compute nodes get the hp repos configured when they are kickstarted, and so they do not need an additional specification of the repo, i.e., the installation commands work as they were before

Reply:
I will take a look at this soon.

Reply:
Checked the conditions and made some changes. Can you confirm the role can be run on your nodes with these changes?